docs/tutorial.rst

```diff
@@ -1000,6 +1000,32 @@ separately from Zarr.
 
 .. _tutorial_copy:
 
+Accessing Zip Files on S3
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The built-in `ZipStore` will only work with paths on the local file-system, however
+it is also possible to access ``.zarr.zip`` data on the cloud. Here is an example of
```

Comment:
I worry that this framing implies that there is something special about the ``.zarr.zip`` string at the end of a filename.  Could we adjust the language to  say "zipped zarr" or something that doesn't assume a particular filename convention?

Reply:
Pushed.